content/chat/setup.textile

```diff
@@ -81,7 +83,32 @@ blang[javascript].
   </script>
   ```
 
-blang[react].
+blang[react,swift].
+
+blang[swift].
+  h3(#spm). Swift Package Manager
+
+  The SDK is distributed as a Swift Package and can be installed using Xcode or by adding it as a dependency in your package’s @Package.swift@.
+
+  Please note the version listed in the language dropdown selector on this page corresponds to the Ably Pub/Sub SDK version, and not the Chat SDK version. Information on the latest Chat SDK release can be found at "https://github.com/ably/ably-chat-swift":https://github.com/ably/ably-chat-swift.
```

Comment:
This will be fixed within the next few weeks, so can be removed. It's not something we state in any other product or language currently.

Reply:
ea58a6b92802deb39dda5fb466323c5393d30bde

Reply:
removed